pom.xml

```diff
@@ -4,8 +4,41 @@
     <artifactId>velocloud-plugin-parent</artifactId>
     <groupId>org.opennms.plugins.velocloud</groupId>
     <name>OpenNMS :: Plugins :: Velocloud Intergration :: Parent</name>
+    <url>https://github.com/OpenNMS/opennms-velocloud-plugin</url>
+    <description>Velocloud Intergration Plugin for OpenNMS</description>
     <packaging>pom</packaging>
 
+    <licenses>
+        <license>
+            <name>GNU Affero General Public License</name>
+            <url>http://www.gnu.org/licenses/agpl.html</url>
+        </license>
+    </licenses>
+
+    <scm>
+        <url>https://github.com/OpenNMS/opennms-velocloud-plugin</url>
+        <connection>scm:git:git@github.com:OpenNMS/opennms-velocloud-plugin.git</connection>
+        <tag>HEAD</tag>
+    </scm>
+
+    <developers>
+        <developer>
```

Comment:
Please include all contributors here, see https://github.com/OpenNMS/opennms-velocloud-plugin/graphs/contributors

Reply:
IMO, we should not list anything here at all. The `developers` tag is meant to list responsible persons which are about to contact about the project. We have other mechanisms to handle such situations. I see them contrary to our project management methods that we have applied.

Reply:
We don't necessary need to list all the developers. We need to have at least 1 developer listed (who might get contacted about the project,) to satisfy the requirements should we decide to publish this plugin to Maven Central.

Reply:
Ok. Got it.